### ISSUE TITLE
Vesicoureteral Reflux Grade V

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6829]
**Preferred term label:**
Vesicoureteral Reflux Grade V

**Synonyms**
VUR grade V, VUR V

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux with massive dilation of the collecting system and severe tortuosity of the ureter.
PMID: 3975102, 18322164

**Parent term (use hpo.jax.org/app)**
High-grade vesicoureteral reflux

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
